variables.md

```diff
@@ -28,6 +28,9 @@ CHECK_RELEASENOTES | boolean | false | Loads `installation/releasenotes` test mo
 CHECK_RELEASENOTES_ORIGIN | boolean | false | Loads `installation/releasenotes_origin` test module.
 CHECKSUM_* | string | | SHA256 checksum of the * medium. E.g. CHECKSUM_ISO_1 for ISO_1.
 CHECKSUM_FAILED | string | | Variable is set if checksum of installation medium fails to visualize error in the test module and not just put this information in the autoinst log file.
+CLUSTER_TYPES | string | false | Set the type of cluster that have to be analyzed (example: "drbd hana"). This variable belongs to PUBLIC_CLOUD_.
+COMMAND_FILE | string | | The LTP test command file (e.g. syscalls, cve)
```

Comment:
I wonder if we should rename this var to `LTP_COMMAND_FILE`

Reply:
out of my head, I had problems when using `LTP_COMMAND_FILE` as it then added the `kernel/run_ltp.pm` test modules. But this is already 3 years ago, so maybe things changed!

Reply:
Can you guys create separate poo# for that?

Reply:
https://progress.opensuse.org/issues/98334